### ISSUE TITLE
page number update

### ISSUE THREAD

**Issue by alvar42** [dryoo/numberof#2]
Could you change the plugin so that a dynamic update of the pagenumber was possible?

Cheers
::: Alex

**Comment by TheManchineel** [comment]
Yeah, can confirm that currently the DokuWiki ```~~NOCACHE~~``` syntax isn't working.
